index.html

```diff
@@ -4,18 +4,169 @@
+                        Phasellus ac tincidunt ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna
+                        elit. Aliquam augue nulla, laoreet sit amet semper vitae, tincidunt dignissim nisl.
+                        Phasellus in justo commodo, tempor est vitae, dictum nibh. Suspendisse cursus scelerisque
+                        magna eu mollis. Integer vitae velit vel tortor aliquam eleifend ut non nisl. Praesent quis
+                        dui nec odio molestie luctus. In gravida risus sed bibendum ultricies. Ut interdum blandit
+                        varius. Aenean porttitor tellus a porta pretium. </p>
+                </div>
+            </div>
+            <div class="article-div">
+                <div class="text-div">
+                    <h1>Lorem ipsum dolor sit amet</h1>
+
+                    <h3>By: Daniel Matheus</h3>
+
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In faucibus sit amet tellus nec
+                        efficitur. Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus.
+                        Phasellus ac tincidunt ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna
+                        elit. Aliquam augue nulla, laoreet sit amet semper vitae, tincidunt dignissim nisl.
+                        Phasellus in justo commodo, tempor est vitae, dictum nibh. Suspendisse cursus scelerisque
+                        magna eu mollis. Integer vitae velit vel tortor aliquam eleifend ut non nisl. Praesent quis
+                        dui nec odio molestie luctus. In gravida risus sed bibendum ultricies. Ut interdum blandit
+                        varius. Aenean porttitor tellus a porta pretium. </p>
+                </div>
+            </div>
+            <div class="article-div">
+                <div class="text-div">
+                    <h1>Lorem ipsum dolor sit amet</h1>
+
+                    <h3>By: Daniel Matheus</h3>
+
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In faucibus sit amet tellus nec
+                        efficitur. Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus.
+                        Phasellus ac tincidunt ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna
+                        elit. Aliquam augue nulla, laoreet sit amet semper vitae, tincidunt dignissim nisl.
+                        Phasellus in justo commodo, tempor est vitae, dictum nibh. Suspendisse cursus scelerisque
+                        magna eu mollis. Integer vitae velit vel tortor aliquam eleifend ut non nisl. Praesent quis
+                        dui nec odio molestie luctus. In gravida risus sed bibendum ultricies. Ut interdum blandit
+                        varius. Aenean porttitor tellus a porta pretium. </p>
+                </div>
+            </div>
+        </div>
+
     </main>
 
     <footer>
```

Comment:
Your `<footer>` tag appears to be completely empty, the guidelines for creating the footer section are the following:
* [ ] The links in the footer should cover the central part of the footer, both horizontally and vertically
* [ ] It should have an unorganized list `<ul>`
* [ ] The links in the footer should be list items `<li>` positioned using the `float` or `flex` display property